examples/mugen/test/retrieval/test_s3d.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import importlib
+
+import pytest
+import torch
+from examples.mugen.retrieval.s3d import BasicConv3d, S3D, SepConv3d
+from test.test_utils import assert_expected, set_rng_seed
+
+
+class TestS3D:
+    @pytest.fixture
```

Comment:
add autouse?

Reply:
i still have to request the fixture to get the returned functions but i added autouse to a new fixture that only calls set_rng_seed